### ISSUE TITLE
Missing libmysqlclient.so.18 fixed by installing additional package

### ISSUE THREAD

**Issue by rsignell-usgs** [conda-forge/wgrib2-feedstock#1]
I did: 
```
conda install wgrib2
```
but `wgrib2` was missing a library:
```
(IOOS) rsignell@js-170-55:~$ wgrib2
wgrib2: error while loading shared libraries: libmysqlclient.so.18: cannot open shared object file: No such file or directory
```
by googling, I found I was able to fix this problem by doing:
```
(IOOS) rsignell@js-170-55:~$ conda install mysql-connector-c -y
Fetching package metadata .............
Solving package specifications: .

Package plan for installation in environment /home/rsignell/miniconda3/envs/IOOS:

The following NEW packages will be INSTALLED:

    mysql-connector-c: 6.1.6-0 conda-forge
```
Then `wgrib2` worked fine:
```
(IOOS) rsignell@js-170-55:~$ wgrib2
wgrib2 v0.2.0.5 7/2016  Wesley Ebisuzaki, Reinoud Bokhorst, John Howard, Jaakko Hyvätti, Dusan Jovic, Daniel Lee, Kristian Nilssen, Karl Pfeiffer, Pablo Romero, Manfred Schwarb, Gregor Schee, Arlindo da Silva, Niklas Sondell, Sam Trahan, Sergey Varlamov

 -0xSec           inv  X      Hex dump of section X (0..8)
 -aerosol_size    inv         optical properties of an aerosol
 -aerosol_wavelength inv         optical properties of an aerosol
 -bitmap          inv         bitmap mode
 -center          inv         center
 -checksum        inv  X      CRC checksum of section X (0..8), whole message (X = -1/message) or (X=data)
...
```
I wasn't sure how to submit a PR to fix the feedstock, so simply reporting the issue here.

**Comment by ocefpaf** [comment]
Looks like we need to pin to `mysql 5.5.24` or use the new packages. I'll take a look at this.

**Comment by ocefpaf** [comment]
Fixed in #2
